pkgs/by-name/ly/ly/package.nix

```diff
@@ -5,27 +5,27 @@
   linux-pam,
   libxcb,
   makeBinaryWrapper,
-  zig_0_14,
+  zig_0_15,
   callPackage,
   nixosTests,
   x11Support ? true,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "ly";
-  version = "1.1.2";
+  version = "1.2.0";
 
   src = fetchFromGitea {
     domain = "codeberg.org";
     owner = "AnErrupTion";
```

Comment:
redirects to https://codeberg.org/fairyglade/ly, could you update here and in the `meta.homepage`

Reply:
You need to squash your commits into one. `ly: 1.1.2 -> 1.2.0`